### ISSUE TITLE
Mermaid diagrams: text labels overflow since 9.0.13

### ISSUE THREAD

**Issue by frankie567** [squidfunk/mkdocs-material#5168]
### Bug description

Labels in nodes and arrows are overflowing the size of their nodes.

<img width="846" alt="Capture d’écran 2023-03-08 à 14 26 12" src="https://user-images.githubusercontent.com/1144727/223724993-facb2c1f-9901-4726-bcf0-b4a991be8e63.png">

It happens since version [9.0.13](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.13). More specifically, it has probably been introduced by this commit: https://github.com/squidfunk/mkdocs-material/commit/5065bf9cd3b55f14a1770b99433385b8fe8cb155

Here is how it looked like in 9.0.12 and before:

<img width="918" alt="Capture d’écran 2023-03-08 à 14 35 21" src="https://user-images.githubusercontent.com/1144727/223726797-dc6bb1e0-9259-422a-99fd-4d2c6653795c.png">

## Suspected root cause

The variable `--md-mermaid-font-family` is now only defined under the `.mermaid` class and not in `:root` anymore. For some reasons, Mermaid compiler may not be able to pick it and thus compute the size of the nodes with its default font instead.

## Workaround

If I manually set the variable in `:root` in custom CSS, it solves the issue:

```css
:root > * {
  --md-mermaid-font-family: var(--md-text-font-family), sans-serif;
}
```

### Related links

- https://squidfunk.github.io/mkdocs-material/reference/diagrams/?h=mermaid#configuration

### Reproduction

[example.zip](https://github.com/squidfunk/mkdocs-material/files/10920990/example.zip)

### Steps to reproduce

1. Enable Mermaid superfence extension.
2. Add a Mermaid diagram with long labels on nodes and arrows.

### Browser

Chrome, Safari, Firefox

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by frankie567** [comment]
Thank you for the super quick feedback and fix 🙏

**Comment by squidfunk** [comment]
Released as part of 9.1.2.
